src/main/java/de/tum/cit/aet/evaluation/domain/InternalComment.java

```diff
@@ -0,0 +1,29 @@
+package de.tum.cit.aet.evaluation.domain;
+
+import de.tum.cit.aet.application.domain.Application;
+import de.tum.cit.aet.usermanagement.domain.User;
+import jakarta.persistence.*;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "internal_comments")
+public class InternalComment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID internalCommentId;
+
+    @ManyToOne
+    @JoinColumn(name = "application")
```

Comment:
(nit pick) I suggest calling it `application_id` to stay consistent with the rest of the naming schemes of the other join columns
```suggestion
    @JoinColumn(name = "application_id")
```

Reply:
I agree with Sarah